### ISSUE TITLE
`helm --namespace` support doesn't actually define the namespace

### ISSUE THREAD

**Issue by maiamcc** [windmilleng/tilt#2415]
Was digging into #2414 and realized that [we DO in fact have support for `--namespace` in the code](https://github.com/windmilleng/tilt/blob/29ad19f92632fa87b55a20ffffd277c85334a8f9/internal/tiltfile/files.go#L220); however, we inject `namespace: foo` for every resource in the generated yaml, but doesn’t actually inject yaml that defines the namespace. Thus if you use the `helm.namespace` arg and the namespace doesn't yet exist, you get an error:
```
got unexpected error during build/deploy: kubectl apply:
stderr: Error from server (NotFound): error when creating "STDIN": namespaces "beepboop" not found
Error from server (NotFound): error when creating "STDIN": namespaces "beepboop" not found
: exit status 1
```

repro:
- in [`tft/snapshots_frontend`](https://github.com/windmilleng/tft/tree/master/snapshot_frontend), create a new Tiltfile:
```
yaml = helm('./chart', namespace='beepboop')
print('~~~~~\n'+str(yaml)+'\n~~~~~')
k8s_yaml(yaml)
```
- `tilt up`, and observe the above "namespaces 'beepboop' not found" erorr
- if you remove the `namespace` arg, the yaml applies without error.

**Comment by nicks** [comment]
looks like this got fixed in #2418

**Comment by maiamcc** [comment]
Ah yep, was waiting until it went out in a release to follow up.
